deployment/CP04OSSM_Deploy.csv

```diff
@@ -1,280 +1,311 @@
+AR-04,,AR-08B,,CP04OSSM-SBD11-06-METBKA000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,CGINS-METLGR-00015,39.94003,-70.87743,,-3,446,
+AR-04,,AR-08B,,CP04OSSM-SBD12-00-DCLENG000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,CGCON-BDCL12-50001,39.94003,-70.87743,,0,446,
+AR-04,,AR-08B,,CP04OSSM-SBD12-03-HYDGN0000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,OL000274,39.94003,-70.87743,,0,446,
+AR-04,,AR-08B,,CP04OSSM-SBD12-04-PCO2AA000,4,1,2016-05-27T15:50:00,2016-10-12T17:12:00,CGMCP-04OSSM-00004,,CGINS-PCO2AA-33183,39.94003,-70.87743,,0,446,
+AR-08B,,AR-18,,CP04OSSM-MFC31-00-CPMENG000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGCON-MCPM03-00007,39.93707,-70.88697,,450,450,
+AR-08B,,AR-18,,CP04OSSM-MFD35-00-DCLENG000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGCON-MDCL35-00003,39.93707,-70.88697,,450,450,
+AR-08B,,AR-18,,CP04OSSM-MFD35-01-ADCPSJ000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGINS-ADCPSJ-21496,39.93707,-70.88697,,450,450,
+AR-08B,,AR-18,,CP04OSSM-MFD35-02-PRESFC000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGINS-PRESFC-01400,39.93707,-70.88697,,450,450,
+AR-08B,,AR-18,,CP04OSSM-MFD35-04-VELPTB000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGINS-VELPTB-12233,39.93707,-70.88697,,450,450,
+AR-08B,,AR-18,,CP04OSSM-MFD35-05-PCO2WB000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGINS-PCO2WB-C0134,39.93707,-70.88697,,450,450,
+AR-08B,,AR-18,,CP04OSSM-MFD35-06-PHSEND000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGINS-PHSEND-P0158,39.93707,-70.88697,,450,450,
+AR-08B,,AR-18,,CP04OSSM-MFD37-00-DCLENG000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGCON-MDCL37-00006,39.93707,-70.88697,,450,450,
+AR-08B,,AR-18,,CP04OSSM-MFD37-01-OPTAAD000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGINS-OPTAAD-00206,39.93707,-70.88697,,450,450,
+AR-08B,,AR-18,,CP04OSSM-MFD37-03-CTDBPE000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGINS-CTDBPE-50004,39.93707,-70.88697,,450,450,
+AR-08B,,AR-18,,CP04OSSM-MFD37-04-DOSTAD000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGINS-DOSTAD-00508,39.93707,-70.88697,,450,450,
+AR-08B,,AR-18,,CP04OSSM-MFD37-07-ZPLSCC000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGINS-ZPLSCC-55101,39.93707,-70.88697,,450,450,
+AR-08B,,AR-18,,CP04OSSM-RIC21-00-CPMENG000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGCON-NCPM02-00005,39.93707,-70.88697,,7,450,
+AR-08B,,AR-18,,CP04OSSM-RID26-00-DCLENG000,5,1,2016-10-12T14:26:00,2017-06-08T12:05:00,CGMCP-04OSSM-00005,,CGCON-NDCL26-00024,39.93707,-70.88697,,7,450,
+AR-08B,,AR-18,,CP04OSSM-RID26-04-VELPTA000,5,1,2016-10-12T14:26:00,201
```

Comment:
Lon for all `CGMCP-04OSSM-00010` entries needs updating to `-70.87742`

Reply:
All Lon have been corrected.